src/frontend/magnify/src/services/http/HttpService.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { rest } from 'msw';
+import { server } from '../../mocks/server';
+import { UsersRoutes } from '../../utils/routes/api/users.routes';
+import { buildApiUrl, HttpService, MagnifyApi } from './http.service';
+
+describe('HttpService', () => {
+  const testingRoute = '/test/http-service/';
+  beforeEach(() => {
+    server.use(
+      rest.get(buildApiUrl(testingRoute), (req, res, ctx) => {
+        if (req.headers.get('Authorization') === 'Bearer null') {
+          return res(ctx.status(401));
+        }
+        return res(ctx.body('Hello'));
+      }),
+    );
+  });
+  it('Testing a request from an offline user', async () => {
```

Comment:
You should name the tests as if they were plain english beginning with the word `it`, in this this case it would be `it tests ..` to be correct english.

( IMO I would use write `it sends a request as a logged-out user` because we already know we are writing a test, and offline can be understood as `off interne connection` ) 

The same goes for other `it()` calls